### PR TITLE
Feature/tune 72 canned responses

### PR DESCRIPTION
**Pull request by maltem-za**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,5 +1,8 @@
 CHANGELOG
 =========
+0.3
+---
+- Add canned responses
```
Comment (smarzban) [inline CHANGES.rst]: @maltem-za can we add the change log when we do the release, and remove it from this PR

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.2.12
+0.3
```
Comment (smarzban) [inline VERSION]: @maltem-za same here, lets change the version number in the release

**Comment by smarzban** [comment]
@maltem-za Left two minor comments. And can you please add some tests

**Comment by maltem-za** [comment]
@saeedmb8 Done as requested.

**Comment by smarzban** [comment]
:+1: :cake:

**Pull request merged**